backends/events/maemosdl/maemosdl-events.cpp

```diff
@@ -51,9 +51,6 @@ static const KeymapEntry keymapEntries[] = {
 
 bool MaemoSdlEventSource::remapKey(SDL_Event &ev, Common::Event &event) {
 
-	Model model = Model(((OSystem_SDL_Maemo *)g_system)->getModel());
-	debug(10, "Model: %s %u %s %s", model.hwId, model.modelType, model.hwAlias, model.hasHwKeyboard ? "true" : "false");
-
```

Comment:
Should this commit be part of this PR or is it a leftover?

Reply:
It's meant to be part of this PR, since it would potentially result in a `cannot convert from pointer to base class 'OSystem' to pointer to derived class 'OSystem_SDL_Maemo' via virtual base 'BaseBackend'` error due to the C-style cast, and since the debug statement doesn't need to be `MaemoSdlEventSource` any more, I decided to just remove it instead of using `dynamic_cast`.